Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -110,7 +110,7 @@
 
 - type: cargoBounty
   id: BountyCrayon
-  reward: 4000
+  reward: 800
```

Comment:
An art supplies crate costs 500, and you need 3 to fufill the crayon bounty, since you can't make crayons on the station, this would mean crayons would be skipped literally every time past the first time it appeared, or cargo would be forced to take a loss.

Reply:
> An art supplies crate costs 500, and you need 3 to fufill the crayon bounty, since you can't make crayons on the station, this would mean crayons would be skipped literally every time past the first time it appeared, or cargo would be forced to take a loss.

You can find crayons on station placed by mappers, you can find it in happy honks also, whats the issue?

Reply:
> > An art supplies crate costs 500, and you need 3 to fufill the crayon bounty, since you can't make crayons on the station, this would mean crayons would be skipped literally every time past the first time it appeared, or cargo would be forced to take a loss.
> 
> You can find crayons on station placed by mappers, you can find it in happy honks also, whats the issue?

Cargo is supposed to be supplying station and working with departments to make a profit. Robbing station of resources can get you banned or so the rules say(or at very least used to before rewrite). This makes you basically rob station for pennies or outright disregard and other many similiar bounties in favor of alternative money sources that are not bounties.